tests/integration/targets/backup_plan/tasks/main.yml

```diff
@@ -113,13 +113,11 @@
             completion_window_minutes: 150
             lifecycle:
               move_to_cold_storage_after_days: 30
-              delete_after_days: 120
```

Comment:
Why was this option removed?

Reply:
I would prefer to add an additional test rather than deleting options from an actual one. This way we keep testing all the lifecycle options and when some of them are not set.